README.md

```diff
@@ -1,127 +1,99 @@
-## TiDB Community
+# TiDB Community
 
-Welcome to the TiDB Community! The main objective is to help members of the TiDB community who share similar interests to learn from and collaborate with each other.
+Welcome to the TiDB Community! The main objective is to help members of the
+TiDB community who share similar interests to learn from and collaborate with
+each other.
 
-Your journey of becoming a contributor and committer starts from here: improving docs, improving code, giving talks, organizing meetups, etc.
+Your journey of becoming a contributor and committer starts from here:
+improving docs, improving code, giving talks, organizing meetups, etc.
 
-## Communicating
-
-You can reach out to the community members via any one of the following ways: 
-
-+ Slack: [https://pingcap.com/tidbslack/](https://pingcap.com/tidbslack/)
-
-+ Google Groups: [https://groups.google.com/forum/#!forum/tidb-user](https://groups.google.com/forum/#!forum/tidb-user)
-
-+ Stack Overflow: [https://stackoverflow.com/questions/tagged/tidb](https://stackoverflow.com/questions/tagged/tidb)
-
-+ Twitter: [https://twitter.com/PingCAP](https://twitter.com/PingCAP)
-
-+ Facebook: [https://www.facebook.com/pingcap2015](https://www.facebook.com/pingcap2015)
-
-
-+ Reddit: [https://www.reddit.com/r/TiDB](https://www.reddit.com/r/TiDB)
-
-## How Can I Help?
```

Comment:
I recommend keeping this part because this can give the audience a direction of how to engage when they first come.